csvqb/csvqb/writers/qbwriter.py

```diff
@@ -421,3 +422,25 @@ def _get_measure_uri(measure: QbMeasure) -> str:
         return _doc_rel_uri(f"measure/{measure.uri_safe_identifier}")
     else:
         raise Exception(f"Unmatched unit type {type(unit)}")
+
+
+def _get_about_url(cube: Cube) -> str:
+    # Todo: Dimensions are currently appended in the order in which the appear in the cube.
+    #       We may want to alter this in the future so that the ordering is from
+    #       least entropic dimension -> most entropic.
+    #       E.g. http://base-uri/observations/male/1996/all-males-1996
+    multi_measure_cols = []
+    aboutUrl = "#obs"
+    for c in cube.columns:
+        if isinstance(c, QbColumn):
+            c: QbColumn
+            if isinstance(c.component, QbDimension):
+                aboutUrl = aboutUrl + f"/{{+{c.uri_safe_identifier}}}"
```

Comment:
Ah, I see why you have dashes, we need to use the [csvw_column_name_safe](https://github.com/GSS-Cogs/csvwlib/blob/7f8ce259d2dc04f3b8dec9aae1a9c20de0f0ce2a/csvqb/csvqb/writers/qbwriter.py#L258) function to be consistent with the names defined in the CSV-W columns section.